vpr/src/place/place.cpp

```diff
@@ -3046,13 +3050,13 @@ void stop_placement_and_check_breakopints(t_pl_blocks_to_be_moved& blocks_affect
         get_bp_state_globals()->get_glob_breakpoint_state()->from_block = size_t(blocks_affected.moved_blocks[0].block_num);
 
         //check for breakpoints
-        f_place_debug = check_for_breakpoints(true);
-        if (f_place_debug)
+        set_placer_debug(check_for_breakpoints(true)); 
```

Comment:
Seems like you should do something like:
if (check_for_breakpoints (true)) {  // What does the true flag passed in mean?  Comment this.
   set_placer_debug (true);
   ....
else {

Reply:
After reviewing the code, I found that the true flag means that the function is called from placer. I added a comment by this meaning.